phoenix-core/src/main/java/org/apache/phoenix/iterate/ScanningResultIterator.java

```diff
@@ -122,8 +111,7 @@ private void getScanMetrics() {
             changeMetric(scanMetricsHolder.getCountOfRowsFiltered(),
                     scanMetricsMap.get(COUNT_OF_ROWS_FILTERED_KEY_METRIC_NAME));
 
-            GLOBAL_SCAN_BYTES.update(scanMetricsMap.get(GLOBAL_BYTES_IN_RESULTS_METRIC_NAME));
-
+            GLOBAL_SCAN_BYTES.update(scanMetricsMap.get(BYTES_IN_RESULTS_METRIC_NAME));
```

Comment:
We should also check that the return of the `get` is not null before passing it to `update`